test/common/modules/client_module.js

```
@@ -35,10 +35,18 @@ define(['ably', 'globals', 'test/common/modules/testapp_module'], function (Ably
     return new Ably.Realtime(ablyClientOptions(helper, options));
   }
 
+  function ablyRealtimeWithoutEndpoint(helper, options) {
+    helper = helper.addingHelperFunction('ablyRealtime');
+    const clientOptions = ablyClientOptions(helper, options);
+    delete clientOptions.endpoint;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)